CHANGELOG.rst

```diff
@@ -19,6 +19,7 @@ Changed
   default timeout of 600 sec which is defined by the python_script action runner (improvement) #5084
 
   Contributed by @hnanchahal
+* Upgrade virtualenv version to latest - 20.2.1
```

Comment:
Can you please follow the Changelog format that's already used above?
Eg. please include that it's (improvement) and PR number.

You're also welcome to include your name or github username as a contributor